### PR TITLE
Visually fix the focus indicator by adjusting padding

### PR DESCRIPTION
**Pull request by dougwaldron**
When navigating a page by keyboard, the browser draws an outline around the currently focused link. When an AnchorJS link is focused, the outlined icon looks awkward because the padding applied is unbalanced.

This pull request adjusts the left and right padding to be equal and also adjusts the margin to maintain the default icon placement. Tests have been updated to match the changes, and all tests are passing. Several of the examples in the documentation also had to be fixed.

Here are screenshots of the focus indicator before and after these changes:

### Before

image image

### After

image image

### CODE REVIEW

**Comment by bryanbraun** [comment]
Ooh, very interesting! This will take a bit of testing but I think it's a great improvement and I'm looking forward to getting this fixed.

I did some small docs updates that unfortunately caused some conflicts. If you have some time to resolve them, that would be helpful. Otherwise, I may be able to take a pass at it next week.

File: docs/index.html [hunk 1]
```diff
@@ -540,17 +541,18 @@ <h3>Link w/CSS Styling</h3>
...
   margin-left: -1.4em !important;
+  padding: 0 !important;
 }
 .anchorjs-link::after {
-  margin-left: 7px;
-  margin-top: -4px;
-  display: block;
+  position: absolute;
+  top: -5px;
+  left: 5px;
```
Comment (bryanbraun) [inline docs/index.html:555]: Wow, thanks for catching this one that I overlooked in my last commit!

**Review by bryanbraun — APPROVED** [review]
I did a lot of checking and this is looking great. Merging!

**Comment by dougwaldron** [comment]
> I did a lot of checking and this is looking great. Merging!

Cool, thanks!

**Pull request merged**